sink/sink

```diff
@@ -592,37 +576,39 @@ def main():
     # Initialize status reporters
     status = Status(config, identifier)
 
-    log = open("log", "w")
-    sys.stdout = Tee(log, sys.stdout)
-    sys.stderr = Tee(log, sys.stderr)
+    log = open("log", "wb")
+    # treat streams as binary stream, to avoid problems with non-ASCII data
+    global stdout, stderr
```

Comment:
ech :/
Why you did not leave it as it was before? That you set up `sys.stderr` and `sys.stdout` and use them everywhere? Does not this work with python3?

Reply:
This leads to all kinds of strange errors, as libraries and other internal code also uses sys.std*. Redefining them to be byte streams instead of a TextWrapper (string streams) wreaks havoc there. Therefore I thought it would be cleaner to have new variables, to point out that these are very much *not* the standard IO streams.

Reply:
@marusak : Does the above explanation make sense, or should I expand further?